components/raftstore/src/store/peer.rs

```diff
@@ -1707,11 +1827,23 @@ impl Peer {
                 return false;
             }
             Ok(RequestPolicy::ReadIndex) => return self.read_index(ctx, req, err_resp, cb),
-            Ok(RequestPolicy::ProposeNormal) => self.propose_normal(ctx, req),
+            Ok(RequestPolicy::ProposeNormal) => {
+                let req_size = req.compute_size();
```

Comment:
Can it hurt performance?

Reply:
I think the resource it costs is much fewer than what we cost in `write_to_bytes`.